### PR TITLE
MarcEspínSanz.GravitonEditor version 2.3.0

### PR DESCRIPTION
**Pull request by oxygen-dioxide**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/24493)

### CODE REVIEW

File: manifests/m/MarcEspínSanz/GravitonEditor/2.3.0/MarcEspínSanz.GravitonEditor.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: MarcEspínSanz.GravitonEditor
```
Comment (ItzLevvie) [inline manifests/m/MarcEspínSanz/GravitonEditor/2.3.0/MarcEspínSanz.GravitonEditor.installer.yaml:4]:
```suggestion
PackageIdentifier: MarcEspinSanz.GravitonEditor
```

File: manifests/m/MarcEspínSanz/GravitonEditor/2.3.0/MarcEspínSanz.GravitonEditor.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: MarcEspínSanz.GravitonEditor
```
Comment (ItzLevvie) [inline manifests/m/MarcEspínSanz/GravitonEditor/2.3.0/MarcEspínSanz.GravitonEditor.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: MarcEspinSanz.GravitonEditor
```

File: manifests/m/MarcEspínSanz/GravitonEditor/2.3.0/MarcEspínSanz.GravitonEditor.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: MarcEspínSanz.GravitonEditor
```
Comment (ItzLevvie) [inline manifests/m/MarcEspínSanz/GravitonEditor/2.3.0/MarcEspínSanz.GravitonEditor.yaml:4]:
```suggestion
PackageIdentifier: MarcEspinSanz.GravitonEditor
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Please update all of the file names from `MarcEspínSanz.GravitonEditor` to `MarcEspinSanz.GravitonEditor`

Also, update the publisher folder name from `MarcEspínSanz` to `MarcEspinSanz`.

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**